infra/testing/sw-env.js

```diff
@@ -24,6 +24,17 @@ const swEnv = makeServiceWorkerEnv();
 // This is needed to ensure new URL('/', location), works.
 swEnv.location = 'https://example.com';
 
+// See https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent/FetchEvent
+class FetchEvent {
```

Comment:
Just put this together for you all. Adds all of the constructors to the global scope which should help for any instanceof checks you do.
[https://github.com/pinterest/service-workers/pull/50](https://github.com/pinterest/service-workers/pull/50)

Reply:
Landed. If you update your service-worker-mock dep to @1.5.0 you can remove this and the manual `swEnv.ServiceWorkerGlobalScope` below.